packages/opentelemetry-exporter-collector/src/platform/node/types.ts

```diff
@@ -24,5 +24,6 @@ import { CollectorExporterConfigBase } from '../../types';
 export interface CollectorExporterNodeConfigBase
   extends CollectorExporterConfigBase {
   keepAlive?: boolean;
+  compress?: boolean;
```

Comment:
I think it would make sense to have different possible compression algorithm in the future so i would prefer to have an enum here like so:
```
enum CompressionAlgorithm {
  NONE = 'none',
  GZIP = 'gzip'
}
```

WDYT ? Otherwise LGTM

Reply:
Yes, it is a good idea. I will change it asap.

Reply:
Sorry, I rebased the branch. In order to review the changes you suggested, please look into the last commit.

Reply:
ltgtm